templates/registration/profile.html

```diff
@@ -23,11 +23,16 @@ <h2 class="profile-header">Welcome, {% firstof user.get_full_name user.get_usern
             </div>
 
             {% if total_earned %}
-                <p class="profile-total">Total earned: ${{ total_earned|floatformat:2|intcomma }}</p>
+                <h5 class="profile-total-header">TOTAL EARNED:</h5>
+                <h5 class="profile-total"> ${{ total_earned|floatformat:2|intcomma }}</h5>
             {% else %}
-                <p class="profile-total">Your money is going towards making some sweet music!</p>
+                <h5 class="profile-total-header">TOTAL EARNED:</h5>
```

Comment:
If both clauses of the `if` statement are going to include TOTAL EARNED, then we should just write that outside of the `if` statement (before the `if`).